HoloRepositoryDemoApplication/Assets/Sample/samplePatients.json

```diff
@@ -1,153 +1,35 @@
-[
-  {
-    "pid": "5d1bf4f17322a9283482fe7e",
-    "name": {
-      "title": "Mrs",
-      "full": "Beverly Cole",
-      "first": "Beverly",
-      "last": "Cole"
-    },
-    "gender": "female",
-    "email": "beverly.cole@miraclis.name",
-    "phone": "+1 (951) 427-3709",
-    "birthDate": "1989-07-07T00:00:00.000Z",
-    "pictureUrl": "http://placehold.it/64x64",
-    "address": {
-      "street": "610 Clark Street",
-      "city": "Linganore",
-      "state": "Texas",
-      "postcode": 9622
-    },
-    "imagingStudySeries": [],
-    "holograms": []
-  },
-  {
-    "pid": "5d1bf4f14c342b1d560679e6",
-    "name": {
-      "title": "Mr",
-      "full": "Roy Santana",
-      "first": "Roy",
-      "last": "Santana"
-    },
+{
+  "additionalProp1": {
```

Comment:
I know it's just the sample patients but I am confused by "additionalProp"? This should be the key, which should equal the pid, if you want to reproduce the API response?

Reply:
I just copied the json data from [API specification](https://app.swaggerhub.com/apis/boonwj/HoloRepository/1.0.0), and "additionalProp" is equals to pid in this situation

Reply:
Oh, I see. Wenjie probably chose this generic name because (at least for the /holograms endpoint) the depending on the query parameters, the result is either nested by pid or by hid. But from your point of view, the response you get really looks as simple as this:
```
{
  "123": {
    "pid": "123",
    ...
  },
  "anotherid": {
    "pid": "anotherid",
    ...
  }
}
```
@boonwj Please confirm

Reply:
Sorry it's a limitation on the openapi spec for expressing dynamic properties. I am unable to change the name of `additionalProperties` to be more expressive. It is used in the 3 cases of mass queries (authors, patients and holograms), in all those cases, the additionalProperties are the IDs that were sent for querying. 

I have since updated the description of the api spec to be more clear about this.